src/controllers/product.controller.js

```diff
@@ -6,4 +6,13 @@ const getNewProducts = catchAsync(async (req, res) => {
   return res.status(200).json(await productService.getNewProducts(orderBy, limitNum));
 });
 
-module.exports = { getNewProducts }
\ No newline at end of file
+const getProduct = catchAsync(async (req, res) => {
+  const {productId} = req.params;
+  if (!productId) {
+    return res.status(400).json({ message: "KEY_ERROR" });
+  }
```

Comment:
```suggestion
  if (!productId) res.status(400).json({ message: "KEY_ERROR" });
```
if의 실행문에서 내부 로직이 지금처럼 거의 없다면 간략하게 one line으로 작성해주시는 것이 깔끔하고 잘 읽힙니다. return keyword도 필요없구요 :)